### ISSUE TITLE
Search bar text animating to center when page loads.

### ISSUE THREAD

**Issue by CooleyGit** [driftyco/ionic#11381]
**Ionic version:**  (check one with "x")
[ ] **1.x**
[ ] **2.x**
[x ] **3.x**

**I'm submitting a ...**  (check one with "x")
[ x] bug report
[ ] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://forum.ionicframework.com/ or http://ionicworldwide.herokuapp.com/

**Current behavior:**

Search bar text animating to center every time page loads. Extremely awkward on flows with search bars on multiple pages.

**Expected behavior:**

The text should start in the center of the input not animate to center. 

**Steps to reproduce:**

Upgrade app from 2.x to 3.x and add ion-searchbar to a page and set animated to true.

**Related code:**

```
<ion-searchbar [animated]="true"></ion-searchbar>
```

**Other information:**

**Ionic info:** (run `ionic info` from a terminal/cmd prompt and paste output below):

```
Cordova CLI: 6.5.0 
Ionic Framework Version: 3.0.1-201704261539
Ionic CLI Version: 2.2.2
Ionic App Lib Version: 2.2.1
Ionic App Scripts Version: 1.3.1
ios-deploy version: 1.8.6 
ios-sim version: 5.0.13 
OS: macOS Sierra
Node Version: v6.9.2
Xcode version: Xcode 8.0 Build version 8A218a
```

**Comment by jgw96** [comment]
Hello! Is this an issue on Ionic 3.1.0 ?

**Comment by CooleyGit** [comment]
Yes I still see it in 3.1.0.

**Comment by CooleyGit** [comment]
This seems to be resolved with nightly "3.1.1-201704282222".
